react/components/ComponentGridItem.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
 
 const ComponentGridItem: FC<Props> = ({ title, description, link }) => (
   <article className="flex flex-column items-between pv4 mh3-l bt b--muted-1">
-    <h3 className="t-heading-4">{title}</h3>
+    <h3 className="t-heading-4 fw1">{title}</h3>
```

Comment:
Change the fontWeight property of the heading-4 in styles config instead of using `t-heading-4 fw1`.

Reply:
Updated it :D